bundles/persistence/org.openhab.persistence.jdbc/README.md

```diff
@@ -50,8 +50,8 @@ This service can be configured in the file `services/jdbc.cfg`.
 | sqltype.ROLLERSHUTTER	    | `TINYINT`        | No | see above |
 | sqltype.STRING		    | `VARCHAR(65500)` | No | see above |
 | sqltype.SWITCH		    | `VARCHAR(6)`     | No | see above |
-| sqltype.TABLEPRIMARYKEY   | `TIMESTAMP`      | No | see above |
-| sqltype.TABLEPRIMARYVALUE | `NOW()`          | No | see above |
+| sqltype.tablePrimaryKey   | `TIMESTAMP`      | No | type of `time` column for newly created item tables |
```

Comment:
These two keys should be in uppercase like all the others.

Reply:
The code expects mixed case. Shall I update all places where those are used to uppercase?

Reply:
I don't see that.

Reply:
e.g. here: https://github.com/openhab/openhab1-addons/blob/master/bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/db/JdbcBaseDAO.java#L321

Reply:
So it is. Thanks.